clang/lib/DPCT/APINames.inc

```diff
@@ -1432,13 +1432,13 @@ ENTRY(__syncthreads_and, __syncthreads_and, true, NO_FLAG, P0, "Successful: DPCT
 ENTRY(__syncthreads_or, __syncthreads_or, true, NO_FLAG, P0, "Successful: DPCT1078")
 ENTRY(tex1Dfetch, tex1Dfetch, true, NO_FLAG, P0, "Successful")
 ENTRY(tex1D, tex1D, true, NO_FLAG, P0, "Successful")
-ENTRY(tex1DLod, tex1DLod, true, NO_FLAG, P4, "DPCT1026/DPCT1027")
+ENTRY(tex1DLod, tex1DLod, true, NO_FLAG, P4, "DPCT1119")
```

Comment:
`ENTRY(tex1DLod, tex1DLod, true, NO_FLAG, P4, "Successful/DPCT1119")`

tex1DLod can be migrated successfully or issue DPCT warning. I suggest we can use this as the comment.
Example: https://github.com/oneapi-src/SYCLomatic/blob/SYCLomatic/clang/lib/DPCT/TypeNames.inc#L138